keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
+
+import tensorflow as tf
+
+from keras_cv.utils import preprocessing
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class RandomResizedCrop(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
+    """Randomly resizes an image then crops to a target size.
+    Args:
+        resize_factor: A tuple of two floats, a single float or `keras_cv.FactorSampler`.
+            `factor` controls the extent to which the image is resized.
+            `factor=0.0` makes this layer perform a no-op operation, while a value of
+            1.0 doubles the size of the image.  If a single float is used, a value
+            between `0.0` and the passed float is sampled.  In order to ensure the value
+            is always the same, please pass a tuple with two identical floats:
+            `(0.5, 0.5)`.
+        crop_size: a tuple of two integers used as the target size to crop images to.
+        interpolation: interpolation method used in the `ImageProjectiveTransformV3` op.
+             Supported values are `"nearest"` and `"bilinear"`.
+             Defaults to `"bilinear"`.
+        aspect_ratio_factor: (Optional) A tuple of two floats, a single float or
```

Comment:
I think aspect ratios respect the aspect ratio convention, 1.0 does not change the image.  aspect_ratio=4/3 shifts aspect ratio by 4/3, etc.

Reply:
As such, we should default this ratio to (1, 1) so it performs a no-op.  I don't think distorting aspect ratio should be a default behavior.

Perhaps we make this argument required?